xc7/utils/prjxray_tile_import.py

```diff
@@ -1447,6 +1449,13 @@ def main():
         '--filter_x', help="Filter imported sites by their x coordinate."
     )
 
+    parser.add_argument(
+        '--no_fasm_prefix',
+        action="store_true",
+        default=False,
```

Comment:
Default is not required with `action="store_true"`, as `store_true` implies default=False

Reply:
Fixed.